sniff.js

```diff
@@ -65,6 +65,11 @@ define(["./features"], function (has) {
 		has.add("mac", dav.indexOf("Macintosh") >= 0);
 
 		has.add("msapp", parseFloat(dua.split("MSAppHost/")[1]) || undefined);
+
+		var wp = parseFloat(dua.split("Windows Phone ")[1]) || undefined;
+		if (wp) {
+			has.add("wp", wp);
+		}
```

Comment:
Could be simplified to just one line:
```
has.add("wp", parseFloat(dua.split("Windows Phone ")[1]) || undefined);
```
but this would lead to an unnecessary entry for "wp" in has' cache when not running on WP, given the current implementation of has.add (https://github.com/ibm-js/requirejs-dplugins/blob/0.4.2/has.js#L54-59). It looks that has.add could avoid adding the entry if the value is undefined. @clmath?